AppCheck/Sources/Core/GACAppCheck.m

```diff
@@ -56,9 +56,9 @@
 
 // TODO(andrewheard): Remove from generic App Check SDK.
 // FIREBASE_APP_CHECK_ONLY_BEGIN
-NSString *const kGACAppCheckTokenAutoRefreshEnabledUserDefaultsPrefix =
+static NSString *const kGACAppCheckTokenAutoRefreshEnabledUserDefaultsPrefix =
```

Comment:
Good catch

Reply:
I have to give credit to the compiler for this one 😛